app/routes/schema-browser.tsx

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { NavLink, Outlet, useLoaderData } from '@remix-run/react'
+import { GridContainer, Icon } from '@trussworks/react-uswds'
+import dirTree from 'directory-tree'
+import { useState } from 'react'
+
+import { SideNav, SideNavSub } from '~/components/SideNav'
+import { feature } from '~/lib/env.server'
+
+// Schema treeItem
+type SchemaTreeItem = {
+  name: string
+  path: string
+  children?: SchemaTreeItem[]
+}
+
+export async function loader() {
+  if (!feature('SCHEMA')) throw new Response(null, { status: 404 })
+  const localDataTree = dirTree(
+    '../../node_modules/@nasa-gcn/schema/gcn/notices'
```

Comment:
Are you sure that this is included in the server bundle? Check the settings in https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/remix.config.mjs.

Reply:
Would that just be an update to the serverDependenciesToBundle var to this `[/^(?!@?aws-sdk\/)/, /^(@nasa-gcn\/schema\/)/]`